Preference Access SSN Matching/REPWRITERSPECS/RB.PREFACCESS.SSNMATCHING

```diff
@@ -0,0 +1,1628 @@
+  DO
+   FILEWRITELINE(BNODEBUGLFNUM,BNODEBUGLFLINE,BNODEBUGLFERROR)
+   IF BNODEBUGLFERROR<>"" THEN
+    DO
+     PRINT "ERROR WRITING TO DEBUG LF "+BNODEBUGLFTITLE+" - "+BNODEBUGLFERROR
+     NEWLINE
+     TERMINATE
+    END
+  END
+ BNODEBUGLFLINE=""
+END [PROCEDURE]
+
+PROCEDURE BNOGETPARAMVAL
+[* Find requested parameter setting and return value.
+**
+**  PASS    BNOPARAMFIND   Targeted parameter setting
+**
+**  RETURN  BNOPARAMFOUND  T/F found flag
+**  RETURN  BNOPARAMVAL    Parameter value found
+*]
+ BNOPARAMFOUND=FALSE
+ FOR BNOPARAMLOOP=1 TO BNOCFGPARAMCOUNT
+  DO
+   IF BNOCFGPARAM(BNOPARAMLOOP)=BNOPARAMFIND THEN
+    DO
+     BNOPARAMVAL=BNOCFGPARAMVAL(BNOPARAMLOOP)
+     BNOPARAMFOUND=TRUE
+     BNOPARAMLOOP=BNOCFGPARAMCOUNT
+    END
+  END
+END [PROCEDURE]
+
+PROCEDURE BNONLTS
+[* Remove leading and trailing spaces
+**  PASS       BNOTEXT
+**  RETURN     BNOTEXT
+*]
+ CALL BNONTS
+ CALL BNONLS
+END [PROCEDURE]
+
+PROCEDURE BNONLS
+[* Remove leading spaces
+**  PASS       BNOTEXT
+**  RETURN     BNOTEXT
+*]
+ IF LENGTH(BNOTEXT)=0 THEN
+  BNOTEXT=""
+ ELSE
+  DO
+   WHILE (LENGTH(BNOTEXT)>0 AND
+          SEGMENT(BNOTEXT,1,1)=" ")
+    DO
+     BNOTEXT=SEGMENT(BNOTEXT,2,LENGTH(BNOTEXT))
+    END
+  END
+END [PROCEDURE]
+
+PROCEDURE BNONTS
+[* Remove trailing spaces
+**  PASS       BNOTEXT
+**  RETURN     BNOTEXT
+*]
+ IF BNOTEXT<>"" THEN
+  BNOTEXT=SEGMENT(BNOTEXT,1,LENGTH(BNOTEXT))
+END [PROCEDURE]
+
+PROCEDURE BNONLZ
+[* Removes leading zeros. Does not remove leading spaces.
+** '00123' results in '123'.
+** ' 0123' results in ' 0123'
+** '00000' results in ''.
+** As a byproduct, trailing spaces are also removed.
+**
+**  PASS       BNOTEXT
+**  RETURN     BNOTEXT
+*]
+ IF BNOTEXT<>"" THEN
+  DO
+   WHILE (SEGMENT(BNOTEXT,1,1)="0")
+    DO
+     IF LENGTH(BNOTEXT)=1 THEN
+      BNOTEXT=""
+     ELSE
+      BNOTEXT=SEGMENT(BNOTEXT,2,LENGTH(BNOTEXT))
+    END
+  END
+END [PROCEDURE]
+
+PROCEDURE VALIDATERANGEINPUT
+[* Validate user input as valid listexpand input. Procedure assumes
+** string is valid until invalid character is found.
+**
+**  PASS     BNOTEXT             Character string to test
+**  RETURN   BNOVALIDRANGEINPUT  Boolean T/F
+*]
+ BNOVALIDRANGEINPUT=TRUE
+ FOR BNOLOCALLOOP=1 TO LENGTH(BNOTEXT)
+  DO
+   IF CHARACTERSEARCH("0123456789,- ALLNONE",UPPERCASE(SEGMENT(BNOTEXT,BNOLOCALLOOP,BNOLOCALLOOP)))=0 THEN
+    DO
+     BNOVALIDRANGEINPUT=FALSE
+     BNOLOCALLOOP=LENGTH(BNOTEXT)
+    END
+  END
+END [PROCEDURE]
+
+PROCEDURE MAINHEADER
+[* Print this report detail on the first page only to assist the user with better interpreting the report
+**
+**                   1         2         3         4         5         6         7         8         9         0         1         2         3
+**          123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012
+*]
+ HEADERS
+  IF FIRSTPAGE=TRUE AND
+     PRINTREPORTHEADER=TRUE THEN
```

Comment:
I don't think you need PRINTREPORTHEADER, as it is always true (unless that is for future use)

Reply:
The option to print or not print the report header is in the setup division.  I didn't see the need to add it to the CFG Letter file but I wanted to make it (somewhat) easier for the CU to turn it off if they desired.
```
SETUP
 FIRSTPAGE                   = TRUE
 PRINTREPORTHEADER           = TRUE
```